cmd/kots/cli/pull.go

```diff
@@ -42,6 +42,7 @@ func PullCmd() *cobra.Command {
 				ExcludeAdminConsole: v.GetBool("exclude-admin-console"),
 				SharedPassword:      v.GetString("shared-password"),
 				CreateAppDir:        true,
+				HelmVersion:         v.GetString("helm-version"),
```

Comment:
is there a way we could use the helm chart spec version everywhere, instead of as a CLI flag?

Reply:
Well if you `kots pull helm://stable/postgres` there is no spec right?

Reply:
Makes sense